lints/lint_subject_common_name_disallowed.go

```diff
@@ -24,7 +24,7 @@ func (l *BadCommonName) Initialize() error {
 }
 
 func (l *BadCommonName) CheckApplies(c *x509.Certificate) bool {
-	return len(c.Subject.CommonName) != 0
+	return len(c.Subject.CommonName) != 0 && util.IsExtInCert(c, util.SANOID)
```

Comment:
This seems like it should actually fail if you consider what the point of this test.